Integrations/DigitalDefense/DigitalDefense.yml

```diff
@@ -0,0 +1,160 @@
+commonfields:
+  id: Digital Defense Inc. FrontlineVM
+  version: -1
+name: Digital Defense Inc. FrontlineVM
+display: Digital Defense Inc. FrontlineVM
+category: Vulnerability Management
+description: Digital Defense Inc.'s Frontline Vulnerability Management solution comprehensively
+  identifies and evaluates the security and business risk postures of network devices
+  and applications deployed as premise, cloud, or hybrid network-based implementations.
+  Now residing entirely in Amazon Web Services (AWS), Frontline VM easily addresses
+  the security compliance requirements of organizations around the globe.
+configuration:
+- display: API Token to access Frontline.Cloud
+  name: apiToken
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+- display: Min vulnerability severity for fetching incidents.
+  name: incidentSeverity
+  defaultvalue: critical
+  type: 15
+  required: false
+  options:
+  - critical
+  - high
+  - medium
+  - low
+  - trivial
+  - info
+- display: Rate at which to check vulnerability events when fetching incidents.
+  name: incidentFrequency
+  defaultvalue: 3 hr
+  type: 15
+  required: true
+  options:
+  - 5 min
+  - 15 min
+  - 30 min
+  - 1 hr
+  - 3 hr
+  - 8 hr
+  - 12 hr
+  - 24 hr
```

Comment:
Fetching incidents directly correlates with how often the user scans their network. Depending how big the network to scan is, the scan may take a couple minutes to an hour. 

The only way a new vulnerability(or incident) would populate, would to perform a new scan. However, we can modify the INCIDENT_FREQUENCY for user input in hours instead of a selection choice of minutes/hours.

Regarding the default frequency of fetching incidents every minute, it would be extremely unnecessary fetching incidents (making a GET request to find new vulnerabilities) every minute since, as described above, newly found vulnerabilities would only populate upon a new scan of the network. Our clients are not scanning their network every minute, therefore fetching incidents (making GET requests to our API every minute) would be unnecessary.